build/stage-update-readme-samples-list.yml

```diff
@@ -0,0 +1,148 @@
+jobs:
+- job: update_readme
+  displayName: 'Keep Samples List in sync in README.md'
+  # TODO: Changes will be made in a following PR to fix the related errors 
+  #       and enable Spell Checking and Markdown Linting
+  # dependsOn: 
+  #   - spell_checking
+  #   - markdown_link
+
+  pool:
+    vmImage: 'ubuntu-latest'
+
+  steps:
+  - checkout: self
+    persistCredentials: true
+
+  - powershell: |
+      # Initialize branchName variable
+      $branchName = ""
+
+      # Check if this is a PR build and set the branch name accordingly
+      if (-not [string]::IsNullOrEmpty($env:SYSTEM_PULLREQUEST_SOURCEBRANCH)) {
+          # For PRs, use the source branch of the PR after stripping the 'refs/heads/' prefix
+          $branchName = $env:SYSTEM_PULLREQUEST_SOURCEBRANCH -replace 'refs/heads/', ''
+      } else {
+          # For direct pushes, use the full branch name from BUILD_SOURCEBRANCH after stripping the 'refs/heads/' prefix
+          $branchName = $env:BUILD_SOURCEBRANCH -replace 'refs/heads/', ''
+      }
+      
+      Write-Host "Operating on branch: $branchName"
+      
+      # Set the variable for subsequent PowerShell scripts
+      Write-Host "##vso[task.setvariable variable=branchName]$branchName"
+    displayName: 'Determine Operation Branch'
+
+  - powershell: |
+      $branchName = $env:BRANCHNAME
+      $changesFound = $false
+
+      # Fetch and configure Git with last editor's details
+      if (-not [string]::IsNullOrEmpty($branchName)) {
+        git fetch --no-tags --prune --depth=1 origin "${branchName}:${branchName}"
+        $lastCommitDetails = git log -1 --pretty=format:"%an <%ae>" -- doc/includes/samples-list.md
+        if (-not [string]::IsNullOrEmpty($lastCommitDetails)) {
+          $name, $email = $lastCommitDetails -split ' <', 2
+          $email = $email.TrimEnd('>')
+          git config --global user.name $name
+          git config --global user.email $email
+          Write-Host "Configured Git to use last editor's name ($name) and email."
+          $changesFound = $true
+        } else {
+          Write-Host "No recent edits found for samples-list.md in the context of the current event."
+          $changesFound = $false
+        }
+      } else {
+        throw "Branch name not determined."
+      }
+      
+      # Output the changesFound variable for other scripts to use
+      Write-Host "##vso[task.setvariable variable=changesFound]$changesFound"
+    displayName: 'Configure Git with Last Editor of samples-list.md'
```

Comment:
@jeromelaban what would make more sense here please?
- Configure Git with Last Editor of samples-list.md
OR
- Using a proper ci user to commit the changes ?
	```
	  git config --global user.email "ci@ci.com"
	  git config --global user.name "ci"
	```

Reply:
let's use devops@platform.uno/Uno Platform Devops